specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/keyvault.json

```
@@ -1731,6 +1731,10 @@
         "id": {
           "type": "string",
           "description": "Full resource id of a vnet subnet, such as '/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/subnet1'."
+        },
+        "ignoreMissingVnetServiceEndpoint": {
+          "type": "boolean",
+          "description": "Property to specify whether NRP will ignore the check if parent subnet has serviceEndpoints configured"
```

Nit: missing punctuation, which most other descriptions have. This also improves the generated SDK documentation.
```suggestion
          "description": "Property to specify whether NRP will ignore the check if parent subnet has serviceEndpoints configured."
```

---

Several other inconsistencies in changes contained within this PR.

---

Sure. I will fix the missing punctuation and space